src/Tools/GenerateSeries/GenerateSeries.py

```diff
@@ -0,0 +1,286 @@
+        return 0.0
+    return ((-1)**(k + 1)*(zeta(k + 1) - 1)/(k + 1)).evalf(decimal_precision)
+
+def digamma_at_2_coefficient(k):
+    if k == 0:
+        return 0.0
+    return ((-1)**(k + 1)*(zeta(k + 1) - 1)).evalf(decimal_precision)
+
+def digamma_asymptotic_coefficient(k):
+    if k == 0:
+        return 0.0
+    return bernoulli(2 * k) / (2 * k)
+
+def trigamma_at_1_coefficient(k):
+    return ((-1)**k * (k + 1) * zeta(k + 2)).evalf(decimal_precision)
+
+def trigamma_asymptotic_coefficient(k):
+    if k == 0:
+        return 0.0
+    return bernoulli(2 * k)
+
+def tetragamma_at_1_coefficient(k):
+    return ((-1)**(k + 1) * (k + 1) * (k + 2) * zeta(k + 3)).evalf(decimal_precision)
+
+def tetragamma_asymptotic_coefficient(k):
+    if k == 0:
+        return 0.0
+    return -(2 * k - 1) * bernoulli(2 * k - 2)
+
+def gammaln_asymptotic_coefficient(k):
+    return bernoulli(2 * k + 2) / (2 * (k + 1) * (2 * k + 1))
+
+def log_1_plus_coefficient(k):
+    if k == S(0):
+        return 0
+    if k % 2 == 0:
+        return S(-1) / k
+    return S(1) / k
+
+def log_1_minus_coefficient(k):
+    if k == 0:
+        return S(0)
+    return S(-1) / k
+
+def x_minus_log_1_plus_coefficient(k):
+    if k <= 1:
+        return S(0)
+    if k % 2 == 0:
+        return S(1) / k
+    return S(-1) / k
+
+def exp_minus_1_coefficient(k):
+    if k == 0:
+        return S(0)
+    return S(1) / factorial(k)
+
+def exp_minus_1_ratio_minus_1_ratio_minus_half_coefficient(k):
+    if k == 0:
+        return S(0)
+    return S(1) / factorial(k + 2)
+
+def get_log_exp_minus_1_ratio_coefficients(count):
+    x = symbols('x')
+    return list(reversed(Poly(log((exp(x) - 1) / x).series(x, 0, count).removeO()).all_coeffs()))
+
+# Formula for mth coefficient of the normcdfln asymptotic:
+# \sum_{n=1}^m (-1)^{n+m+1} / n * \sum_{l1, l2, ..., ln \in N, l1 + l2 + ... + ln = m} (2 * l1 - 1)!! * (2 * l2 - 1)!! * ... * (2 * ln - 1)!!
+# Can be obtained by composing the Taylor expansion for log(1 + x) and asymtotic expansion for erfc
+def normcdfln_asymptotic_coefficient(m):
+    if m == 0:
+        return 0
+    def next(v):
+        idx = 1
+        while idx < len(v) and v[idx] == 0:
+            idx = idx + 1
+        if idx == len(v): return False
+        v0 = v[0]
+        v[0] = 0
+        v[idx] = v[idx] - 1
+        v[idx - 1] = v0 + 1
+        return True
+    result = S((-1)**(2 + m)) * factorial2(2 * m - 1)
+    for n in range(2,m+1):
+        coef = S((-1)**(n + 1 + m)) / n
+        deltas = []
+        for k in range(0, n):
+            deltas.append(0)
+        deltas[-1] = m - n
+        accSum = S(0)
+        while True:
+            accProd = S(1)
+            for delta in deltas:
+                accProd = accProd * factorial2(2 * (delta + 1) - 1)
+            accSum = accSum + accProd
+            if not next(deltas):
+                break
+        result = result + coef * accSum
+    return result
+
+def main():
+    print("(1) - Gamma at 2:")
```

Comment:
Where do the code comments come from?  Why not print those out as well?

Reply:
Printing whole comments now.